lib/widgets/theme.dart

```diff
@@ -14,7 +14,10 @@ ThemeData zulipThemeData(BuildContext context) {
       scrolledUnderElevation: 0,
       backgroundColor: Color(0xfff5f5f5),
 
-      shape: Border(bottom: BorderSide(color: Color(0xffcccccc))),
+      shape: Border(bottom: BorderSide(
+        color: Color(0x33000000),
```

Comment:
Now, the app-bar bottom border is colored by the new Figma's
    `border-bar`, and the main background color is `bg-main`. The app
    bar background is unchanged because it already matches `bg-top-bar`.

This feels like the sort of information that's helpful to have in the code itself, so as a comment something like:

```suggestion
        color: Color(0x33000000), // `border-bar` in Figma
```

As part of #95 I think you'll be turning these into some sort of named reference, so that this would be `foo.borderBarColor` for some `foo`. That'll be another way of putting this information into the code, which is better than a comment. But until then a comment seems helpful.